src/core/Pricing/PricingCards.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { Fragment } from "react";
+import type { PricingDataFeature } from "./types";
+import { determineThemeColor } from "../styles/colors/utils";
+import { ColorClass, Theme } from "../styles/colors/types";
+import Icon from "../Icon";
+import FeaturedLink from "../FeaturedLink";
+import { IconName } from "../Icon/types";
+
+type PricingCardsProps = {
+  data: PricingDataFeature[];
+  theme?: Theme;
+  delimiter?: IconName;
+};
+
+const PricingCards = ({
+  data,
+  theme = "dark",
+  delimiter,
+}: PricingCardsProps) => {
+  // work out a dynamic theme colouring, using dark theme colouring as the base
+  const t = (color: ColorClass) => determineThemeColor("dark", theme, color);
```

Comment:
Any colour class supplied to `t` will be flipped across the palette if the theme isn't the base theme - in this case, "dark"